### PR TITLE
Simplify pull request template.

### PR DESCRIPTION
**Pull request by markdorison**
## Description

Simplify the pull request template while maintaining the overarching goal of making it easy for the author to provide a description that makes it straightforward for someone to review in a timely manner.

## Motivation / Context
Feedback from the team.

## Testing Instructions / How This Has Been Tested
N/A

## Documentation
N/A

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,25 +1,15 @@
 ## Description
-
-<!--- Describe your changes in detail. -->
+<!--- Describe your changes in detail. The sections suggested below  -->
```
Comment (adamzimmermann) [inline .github/PULL_REQUEST_TEMPLATE.md]: Did you forget the end of the sentence?

**Review by adamzimmermann — COMMENTED** [review]
I like it.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (markdorison) [reply .github/PULL_REQUEST_TEMPLATE.md]: Good catch. Updated!

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -1,25 +1,15 @@
...
 
 ## Motivation / Context
 <!--- Why is this change required? What problem does it solve? -->
-<!--- If it fixes, or is related to, an open issue, link to the issue here. -->
+<!--- If it fixes, or is related to an open issue, link to the issue here. -->
```
Comment (agarzola) [inline .github/PULL_REQUEST_TEMPLATE.md]: What do you think about removing that first comma as well? I think it flows a bit better that way.
```suggestion
<!--- If it fixes or is related to an open issue, link to the issue here. -->
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -1,25 +1,16 @@
 ## Description
-
-<!--- Describe your changes in detail. -->
+<!--- Describe your changes in detail. The sections suggested are intended to make -->
+<!--- it easy to create a descriptive PR that is easy to review. Change as needed! -->
```
Comment (agarzola) [inline .github/PULL_REQUEST_TEMPLATE.md]: +++ I love starting off making it explicit that these are just guidelines, inviting PR authors to change the sections as they see fit.

Can we remove the trailing space on this line?
```suggestion
<!--- it easy to create a descriptive PR that is easy to review. Change as needed! -->
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 4]
```diff
@@ -1,25 +1,16 @@
...
-<!--- Do any of the changes affect the accuracy of existing documentation? --->
-<!--- Do they warrant new sections within the current documentation? --->
-
-## PR Author Checklist
-
-- [ ] My PR description includes a reference to a ticket or other "motivation" for these changes.
-- [ ] My PR description includes details of how I tested these changes.
-- [ ] My PR description includes testing instructions for reviewers with links to the testing environment, if applicable.
-- [ ] My code includes documentation updates to existing documentation or new sections if applicable.
+<!--- Did any of the changes cause documentation to need to be updated? --->
```
Comment (agarzola) [inline .github/PULL_REQUEST_TEMPLATE.md]: Can we improve this question a bit?
```suggestion
<!--- Do any of the changes warrant documentation updates? --->
```

Take it to leave it, though. This is not a deal-breaker.

**Review by agarzola — APPROVED** [review]
This is looking great. I have a few suggestions; none of them a deal-breaker, so I’m approving this.

**Review by adamzimmermann — APPROVED** [review]
I like the new suggestions.

**Comment by chrisfree** [comment]
I love these changes. LGreatTM.

**Pull request merged**